routes/sapiV1CapiV2.js

```diff
@@ -155,37 +165,160 @@ function prepAnnotationsGroup( groupName, annosDetails, groupDetails, searchResp
+        ;
+
+        if (uuidsInClique0.length > 0 && uuidsInClique1.length > 0) {
+          foundAStraddler = true;
+          break;
+        } else if (uuidsInClique0.length > 0) {
+          uuidsOfName.forEach(uuid => { clique0KnownUuids[uuid] = true; });
+          clique0Names.push(name);
+        } else if (uuidsInClique1.length > 0) {
+          uuidsOfName.forEach(uuid => { clique1KnownUuids[uuid] = true; });
+          clique1Names.push(name);
+        } else {
+          throw new Error( `prepAnnotationsGroup: should never experience this situation where the name (${name}) has no uuids in both clique0 (${clique0Name}) and clique1 (${clique1Name})`);
+        }
+      }
+
+      if (! foundAStraddler) {
+        debug( `prepAnnotationsGroup: found a clique: clique0Name=${clique0Name}, clique1Name=${clique1Name}` );
+        const clique0NamesWithoutTaxonomy = clique0Names.map( name => { return name.split(':')[1]; });
+        const clique0Uuids = Object.keys(clique0KnownUuids);
+        const clique0NamesWithCountsBR = clique0Names
+        .map( name => {
+          const groupCount = groupDetails.uuidsGroupedByItem[name].length;
+          const cliqueCount = (name === clique0Name)? `${clique0Uuids.length} / ${groupCount}` : `${groupCount}`;
+          return {
+            name : name.split(':')[1],
+            groupCount,
+            cliqueCount,
+          }; })
+        .map( details => { return `${details.name} (${details.cliqueCount})`; })
+        ;
+        const clique0 = {
+          nameWithTaxonomy: clique0Name,
+          name : clique0Name.split(':')[1],
+          namesWithTaxonomy: clique0Names,
+          names : clique0NamesWithoutTaxonomy,
+          namesBR : clique0NamesWithoutTaxonomy.join( ' +<BR>'),
+          namesWithCountsBR : clique0NamesWithCountsBR.join(' +<BR>'),
+          uuids : clique0Uuids,
+        };
+        const clique1NamesWithoutTaxonomy = clique1Names.map( name => { return name.split(':')[1]; });
+        const clique1Uuids = Object.keys(clique1KnownUuids);
+        const mainNameUuidsInClique1 = groupDetails.uuidsGroupedByItem[clique0Name]
+        .filter( uuid => { return clique1KnownUuids.hasOwnProperty(uuid)})
+        ;
+        const clique1NamesWithCountsBR = clique1Names
+        .map( name => {
+          const groupCount = groupDetails.uuidsGroupedByItem[name].length;
+          const cliqueCount = (name === clique0Name)? `${mainNameUuidsInClique1.length} / ${groupCount}` : `${groupCount}`;
+          return {
+            name: name.split(':')[1],
+            groupCount,
+            cliqueCount,
+          }; })
+        .map( details => { return `${details.name} (${details.cliqueCount})`; })
+        ;
+        const clique1 = {
+          nameWithTaxonomy: clique1Name,
+          name: clique1Name.split(':')[1],
+          namesWithTaxonomy: clique1Names,
+          names : clique1NamesWithoutTaxonomy,
+          namesBR: clique1NamesWithoutTaxonomy.join(' +<BR>'),
```

Comment:
Please use lowercase `<br>`s, this is not XHTML!

Reply:
OK.
Sorry.
ok.